ansible/roles/druid-ingestion-health/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+druid_path : "/data/druid"
+druid_overlord_host: 11.2.1.20
```

Comment:
Shouldn't the ip addresses be variabilized?

Reply:
variablized the ip address